src/status_im2/contexts/onboarding/identifiers/profile_card/view.cljs

```diff
@@ -0,0 +1,73 @@
+(ns status-im2.contexts.onboarding.identifiers.profile-card.view
+  (:require [react-native.core :as rn]
+            [react-native.hole-view :as hole-view]
+            [react-native.reanimated :as reanimated]
+            [react-native.fast-image :as fast-image]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.markdown.text :as text]
+            [quo2.components.profile.profile-card.style :as profile-card-style]
+            [quo2.components.avatars.user-avatar.view :as user-avatar]
+            [utils.worklets.identifiers-highlighting :as worklets.identifiers-highlighting]
+            [status-im2.common.resources :as resources]
+            [status-im2.contexts.onboarding.identifiers.profile-card.style :as style]))
+
+(defn- f-profile-card-component
+  [{:keys [profile-picture name hash emoji-hash on-card-press
+           customization-color card-style progress]
+    :or   {customization-color :turquoise
```

Comment:
Hi @J-Son89, Do we have a default color stored somewhere we can use? We are using different value of default color, at different places

Reply:
Yes you're are right, I believe we do have multiple uses of this. We should put it in a constants file imo